docs/installation.md

```diff
@@ -2,11 +2,11 @@
 
 ## Prerequisites
 
-scvi-tools can be installed via conda or pip. If you don't know which to choose, we recommend conda for beginner users.
+scvi-tools can be installed via conda or pip. If you don't know which to choose, we recommend conda.
 
 ### conda prerequisites
 
-1. Install Conda. We typically use the [Miniconda] Python distribution. Use Python version >=3.7.
+1. Install conda. We typically use the [mambaforge] distribution. Use Python version >=3.8, and consider using mamba instead of conda. Mamba is a drop-in replacement for conda that is much faster and more efficient.
```

Comment:
```suggestion
1. Install conda. We typically use the [mambaforge] distribution. Use Python version >=3.8, and consider using mamba instead of conda. Mamba is a drop-in replacement for conda that is significantly more efficient.
```